akd_mysql/src/mysql_storables.rs

```diff
@@ -641,8 +645,16 @@ impl MySqlStorable for DbRecord {
                             val.try_into().map_err(|_| cast_err())?,
                             len,
                         ))),
-                        (Err(_len_err), _) => Err(Error::from("Error decoding length")),
-                        (_, Err(_val_err)) => Err(Error::from("Error decoding value")),
+                        (Err(_len_err), _) => Err(Error::from(mysql_async::ServerError {
```

Comment:
Are these conversions necessary? It looks like we're just converting to the base Error type. Or am I misreading it and it's a MySQL error type?

Reply:
It's a MySQL error type. The problem was that in 0.29, MySQL implemented the From<& static str> trait, but support for that was removed in 0.30 onwards. So now I decided to just the more cumbersome notation here...